extra-libs/libyuv/spec

```diff
@@ -0,0 +1,4 @@
+_COMMIT=6862f18e73191a62b6bc8dda56244e4cae5f192a
+VER=20211221+git${_COMMIT:0:8}
```

Comment:
This versioning rule seems to violate our Styling Manual. See https://wiki.aosc.io/developer/packaging/package-styling-manual/#versioning-variables.